artic/minion_validator.py

```diff
@@ -102,7 +102,6 @@
         {
             241: ['C', 'T', "snp", 1],
             3037: ['C', 'T', "snp", 1],
-            12733: ['C', 'T', "snp", 1],
```

Comment:
This var is encompassed by the line below (due to the x2 in the final field of the tuple) so this line is superfluous and can be removed safely.